src/app/components/core/manage-services/list-manage-services.component.ts

```diff
@@ -0,0 +1,342 @@
+  servicesSchedules = [];
+
+  public reenableButton = new EventEmitter<boolean>(false);
+  @ViewChild(ManageServiceModalComponent, { static: true }) serviceModal: ManageServiceModalComponent;
+
+  constructor(
+    public sharedService: SharedService,
+    private alertService: AlertService,
+    public ngProgress: ProgressBarService,
+    private dialogService: DialogService,
+    public servicesApiService: ServicesApiService,
+    public rolesService: RolesService,
+    public schedulesService: SchedulesService,
+    private response: ResponseHandler
+  ) {}
+
+  ngOnInit() {
+    this.showServices();
+  }
+
+  // sleep time expects milliseconds
+  sleep (time) {
+    return new Promise((resolve) => setTimeout(resolve, time));
+  }
+
+  showServices() {
+    this.getServices();
+    this.getSchedules();
+
+    let callsStack = {
+      installed: this.servicesApiService.getInstalledServices(),
+      available: this.servicesApiService.getAvailableServices()
+    }
+    this.sleep(1000).then(() => {
+      forkJoin(callsStack)
+        .pipe(
+          map((response: any) => {
+            const installed = <Array<any>>response.installed;
+            const available = <Array<any>>response.available;
+            const result: any[] = [];
+            result.push({
+              ...{ 'installed': installed },
+              ...{ 'available': available }
+            });
+            this.getInstalledServices(installed["services"]);
+            this.getAvaiableServices(available["services"]);
+
+            let installedServicePkgsNames = []
+            this.installedServicePkgs.forEach(function(s) {
+              installedServicePkgsNames.push(s["package"]);
+            });
+
+            // Remove service name from available list if it is already installed
+            this.availableServicePkgs = this.availableServicePkgs.filter((s) => !installedServicePkgsNames.includes(s.package));
+            return result;
+          })
+        )
+        .subscribe((result) => {
+          result.forEach((r: any) => {
+            this.ngProgress.done();
+            if (r.failed) {
+              if (r.error.status === 0) {
+                console.log('service down ', r.error);
+              } else {
+                this.alertService.error(r.error.statusText);
+              }
+            } else {
+              this.response.handleResponseMessage(r.type);
+            }
+          });
+        });
+    });
+  }
+
+  public getInstalledServices(services) {   
+    let svcs = services.filter(
+      (s) => !["south", "north", "storage"].includes(s)
+    );
+    this.installedServicePkgs = this.expectedServices.filter(
+      (s) => svcs.includes(s.process)
+    );
+
+    let replacement;
+    let atIndex = -1;
+    this.installedServicePkgs.forEach((installed, idx) => {
+      replacement = structuredClone(installed);
+      let found_svc = this.servicesRegistry.find(s => s.type == installed.type);
+      if (found_svc === undefined){
```

Comment:
better to change variables from snakeCase to camelCase as we follow this practice.